ipc/README.md

```diff
@@ -0,0 +1,88 @@
+# templates-ipc
+
+## Summary
+This Microservice template/example is a project, which showcases how you can create REST endpoints within your Microservice and call another Microservice REST API hosted in Cumulocity (Interprocess communication IPC)
+
+There is one REST endpoint created by this Microservice. This is defined in the classes `DeviceController.class`. This controller defines the individual API of the microservice.
+
+The DeviceController defines one endpoint:
+- `GET /devices/names` 
+
+    will return a list of device names
+    
+
+This DeviceController uses another REST API of another Microservice [basic](../basic) to accomplish the task (response). Instead of using REST templates a declarative way of defining REST clients is used in that example. [Spring cloud - Feign](https://cloud.spring.io/spring-cloud-netflix/multi/multi_spring-cloud-feign.html)
+
+However, this example can also easily changed to rest templates. The integral part of this template you will find in `TemplatesBasicClientConfig`. This configuration contains the request interceptor which handles in that case the authorization (Autorization header) and setting the right base URL. If you want to call another microservice which is not hosted in Cumulocity you can modify the request interceptor to do a OAuth2 authorization for example. [See](https://www.baeldung.com/spring-cloud-feign-oauth-token).
```

Comment:
little wording improvements (even though I'm no native speaker....)
- "also easily changed to rest templates" => "also easily be changed to use RestTemplates"
- "The integral part of this template you will find in `TemplatesBasicClientConfig`" => "You can find the integral part of this template in `TemplatesBasicClientConfig`" (and add a link to the file)

Reply:
done